components/NavItem.tsx

```
@@ -0,0 +1,24 @@
+
+interface NavItemProps {
+  icon: string;
+  label: string;
+  isActive: boolean;
+}
+
+const NavItem: React.FC<NavItemProps> = ({ icon, label, isActive }) => {
+  return (
+    <li className="flex flex-col gap-1 items-center p-2 cursor-pointer">
+      <div className="flex justify-center items-center w-6 h-6">
+        <div dangerouslySetInnerHTML={{ __html: icon }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)